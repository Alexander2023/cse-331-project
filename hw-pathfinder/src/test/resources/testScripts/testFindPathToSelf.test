# Tests that path finding works properly when asked to find the path from a node to itself

CreateGraph g

AddNode g p1

FindPath g p1 p1