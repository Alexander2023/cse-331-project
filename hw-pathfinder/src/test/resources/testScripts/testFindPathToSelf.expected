# Tests that path finding works properly when asked to find the path from a node to itself

created graph g

added node p1 to g

path from p1 to p1:
total cost: 0.000